SNIPS/snip-25.md

```diff
@@ -0,0 +1,267 @@
+---
+snip: 25
+title: Starknet Security Council (SSC)
+author: Henri Lieutaud <@l-henri>, Boaz <@BoazStark>, Elias Tazartes <@Eikix>, Mattéo Georges <@mattegoat>, Patrícia Gil de Brolezzi <@pat_zip>, luiz-lvj <lvj253@gmail.com>, Luca Donno <@lucadonnoh>, Nicolas Consigny <@nconsigny>
+status: Review
+type: Meta
+created: 2024-07-06
+---
+
+## Abstract
+
+This Meta SNIP lays out the structure, motivation and decisions of the Starknet Security Council. It describes the process and rules that surronds the Starknet Security Council.
+
+The Starknet Security Council is made up of 12 members who are both geographically and organizationally diverse. They are tasked with safeguarding the security of Starknet. The Council assesses proposed upgrades to the network’s core contracts and approves them if no security risks are identified. In the event of an emergency, the Council has the authority to expedite necessary actions.
+
+## Motivation
```

Comment:
The whole line 14 appears in section 'Motivation'. Could you delete it from the abstract?

Reply:
Done!